slate-video-library/src/Pages/SingleVideoPage/SingleVideopage.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import Footer from "../../Components/Footer/Footer";
+import Header from "../../Components/Header/Header";
+import Sidebar from "../../Components/Sidebar/Sidebar";
```

Comment:
Suggestion: you can take Footer , Header , Sidebar inside a single folder, where you have all the shared components.This will enchance your folder structure